COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -559,63 +556,39 @@ The COBOL Language Support extension checks for mistakes and errors in COBOL cod
 
 ## Navigation of Code
 
-The Code4z package enables several features for ease of navigation through code.
-
-### Breadcrumb View
-
-The breadcrumb view across the top of the editor shows where the current line of code exists within the structure of the COBOL source code. You can click each element on the bar to see that section of the code highlighted, or to select a code element within the section and navigate to it.
-
-![](Images/code4z-img2.png)
```

Comment:
If this image is no longer referenced anywhere in the course, then we should remove the image from the repo as well.